package.json

```diff
@@ -10,7 +10,7 @@
     "server:init": "next build && pm2 start --name tug-website-next-server npm -- run start",
     "server:restart": "pm2 restart tug-website-next-server",
     "start": "next start",
-    "storybook": "lerna run --scope @pingcap/pingcap-ui storybook -- -- -s $(pwd)/public",
+    "storybook": "lerna run --scope @tidb-community/ui storybook -- -- -s $(pwd)/public",
```

Comment:
As discussed, we will deploy all the community-related packages under the '@tidb-community' org.